src/applications/personalization/rated-disabilities/tests/components/TotalRatingStates.unit.spec.jsx

```diff
@@ -8,19 +8,12 @@ import {
 describe('TotalRatingStates', () => {
   it('should include an error message', () => {
     const message = errorMessage();
-    expect(message.props.headline).to.equal(
-      'We’re sorry. Something went wrong on our end',
-    );
     expect(message.props.status).to.equal('error');
     expect(message.props.isVisible).to.equal(true);
   });
 
   it('should include a no rating message', () => {
     const message = missingTotalMessage();
-
-    expect(message.props.headline).to.equal(
```

Comment:
Removed this check because we're no longer passing `headline` to the `AlertBox`.